src/gurt/dlog.c

```diff
@@ -690,13 +708,22 @@ void d_vlog(int flags, const char *fmt, va_list ap)
 	 * log it to stderr and/or stdout.  skip part one of the header
 	 * if the output channel is a tty
 	 */
+printf(" SAB: d_vlog: flag 0x%x\n",flags);
```

Comment:
(style)  space required after that ',' (ctx:VxV)

Reply:
Print for debug only.  Removed.